server/aws-lsp-partiql/src/server/language-service.ts

```diff
@@ -5,7 +5,7 @@ import { initSync, parse_as_json } from '../partiql-parser-wasm/partiql_playgrou
 import { convertObjectToParserError } from './error-parsing/parser-errors'
 
 export function normalizeQuery(data: string): string {
-    return data != null ? data.replace(/\"/g, '\\"') : ''
```

Comment:
Is there a (now-failing) test case for the escaping of quotes and the situation for which this was necessary?

Reply:
This was copied from the partiql-playground (first party package using the same parser) which had the same code, they also removed it in https://github.com/partiql/partiql-rust-playground/pull/16 after we discussed this bug with the team maintaining that.

There wasn't a test for the escaping before, I added one to verify that quoted identifiers don't give diagnostics.